h2o-core/src/main/java/hex/Model.java

```diff
@@ -42,6 +43,23 @@
   protected ScoringInfo[] scoringInfo;
   public IcedHashMap<Key, String> _toDelete = new IcedHashMap<>();
 
+  public static Model[] fetchAll() {
```

Comment:
Minor: Not bad, but we should have some access utility to do that instead of putting collection code into entity.

Conceptually, entity code (Model) is one class, how to get collection of entities is another class.